### ISSUE TITLE
Feature request: install pkg files directly

### ISSUE THREAD

**Issue by TanyaEleventhGoddess** [blastrock/pkgj#434]
Makes no sense downloading pkg files with the own Vita 12yo slow wifi when you can just download the pkg file with your pc at a way faster speed, plug the console through USB and ram the pkg file inside, do it like 3DS FBI tool which allows both loading files or downloading them from the net.

After all shouldn't be an issue, you can still extract the titleid from the pkg file and then looking up the tsv table for the zRIF

Also, consider the loading method, would be best by using VitaShell or by implementing yourself a Files browser, at worst use a directory for this

**Comment by Masamune3210** [comment]
...if you are downloading the pkg on your computer, you can also extract the pkg on your computer. Just use NPS Browser and it will spit out the decrypted contents, ready to be dragged onto your console's storage

**Comment by TanyaEleventhGoddess** [comment]
> ...if you are downloading the pkg on your computer, you can also extract the pkg on your computer. Just use NPS Browser and it will spit out the decrypted contents, ready to be dragged onto your console's storage

NPS Browser is windows-only, I've tried doing it manually for gravity rush with pkg2zip but the application crashes, downloaded with pkgj works fine

**Comment by Masamune3210** [comment]
https://github.com/JK3Y/NPS-Browser-macOS

**Comment by TanyaEleventhGoddess** [comment]
> https://github.com/JK3Y/NPS-Browser-macOS

I use linux, not mac

**Comment by Masamune3210** [comment]
https://github.com/evertonstz/pynps

**Comment by Masamune3210** [comment]
Man, if only these were more than a simple google search away lol

**Comment by TanyaEleventhGoddess** [comment]
> Man, if only these were more than a simple google search away lol
pynps github isn't exactly the first result google shows you when you google "how to install pkg package on psvita"

I'll try it but would still be nice if pkgj was able to load pkg files directly

**Comment by Masamune3210** [comment]
pkgj hasn't had a new release in almost a year and a half, the chances of a new one coming out are very slim. Hence, the alternative options

**Comment by Masamune3210** [comment]
> > Man, if only these were more than a simple google search away lol
> 
> pynps github isn't exactly the first result google shows you when you google "how to install pkg package on psvita"
> 
> I'll try it but would still be nice if pkgj was able to load pkg files directly

No, but it is the first result when you search "Nps browser linux"

**Comment by TanyaEleventhGoddess** [comment]
> https://github.com/evertonstz/pynps
Tried it right now on Hotline Miami 2


OBVIOUSLY works with pkgj but doesn't with pynps/pkg2zip

**Comment by Masamune3210** [comment]
Then you are missing a step somewhere or something is misconfigured, because I know for a fact that both work perfectly fine

**Comment by TanyaEleventhGoddess** [comment]
> Then you are missing a step somewhere or something is misconfigured, because I know for a fact that both work perfectly fine

Try it yourself with 3.65 firmware then what do I have to say

**Comment by TheDireHavok** [comment]
I have tried over and over again without any luck, to get PSM games to download with any version of PKGj. It gives "error bad http status:404"  Can anyone assist?
